src/public/js/rs.js

```diff
@@ -1,16 +1,33 @@
 'use strict';
 var RS = (function() {
-  // (actually ROT-13 right now, as a dummy example. We can replace w/ RS)
-  var ROT = 13;
+  // Using a RS with 256 symbol size 255 symobls in a codeword and 245 message symbols.
+  // n-k = 10 code symbols. Can correct up to 5 symbol errors in each 255 symbol block
+  var n = 255;
+  var k = 245;
 
   function encode(msg) {
-    // dummy cipher for now -- advances each letter in message by ROT
-    return UTIL.shift(msg, ROT);
+    // Does Reed Solomon decoding on k length message block of symbols.
+    // Cast msg to Uint8Array
+    // If msg is not length k then pad with 0s.
+    var rsMsg = new Uint8Array(k);
+    for (var i = 0; i < msg.length; i++) {
+      rsMsg[i] = msg[i].charCodeAt();
+    }
+    return encodeRSBlock(rsMsg, n, k);
   }
 
   function decode(msg) {
-    // dummy cipher for now -- advances each letter in message by -ROT
-    return UTIL.shift(msg, -ROT);
+    // Decode and correct an n length RS codeword in to a k length message.
+    // Cast input codeword to Uint8Array
+    var decoded = decodeRSBlock(msg, n, k);
+    var decodedString = new Array(k);
+    for (var i = 0; i < k; i++) {
+      if(decoded[i] == 0) {
+        break;
+      }
+      decodedString[i] = String.fromCharCode(decoded[i])
+    }
+    return decodedString
```

Comment:
I decided against this for a couple of up for discussion points! 
1. Encode takes and returns an array, but if I make this change decode will take an array and return a string. I am ok with this since the encode and decode are already kind of wrappers around the encodeRS and decodeRS function, but its something to consider.
2. String concat is much less efficient than array push (assuming as you assumed it doubles the underlying structures size each time, which I think is probably a good bet). Making this change would return a string so in the main function where we collect the blocks, instead of an array push, it would be string concat, which copies all of the contents of both strings and returns a new string each time.